.eslintignore

```diff
@@ -4,3 +4,4 @@ seeders/
 migrations/
 coverage/
 dist/
+test/
```

Comment:
I don't think test should be ignored, it's code.

Reply:
humm ok, will remove it.